node/src/components/consensus/era_supervisor.rs

```diff
@@ -225,6 +225,30 @@ where
         u64::from_le_bytes(result[0..std::mem::size_of::<u64>()].try_into().unwrap())
     }
 
+    /// Returns an iterator over era IDs of `num_eras` past eras, including the provided one.
```

Comment:
This is implemented so it will return `num_eras + 1`.  The comment could maybe be a little less ambiguous if "including" were swapped for "plus" or something like that?